src/lwip/efr32/lwipopts.h

```diff
@@ -94,11 +92,11 @@
 #define LWIP_ND6_MAX_NEIGHBOR_ADVERTISEMENT (0)
 #define LWIP_ND6_TCP_REACHABILITY_HINTS (0)
 
-#ifdef CHIP_CONFIG_EFR32MG21_PBUF_POOLS
+#if defined(EFR32MG21)
 #define MEMP_SEPARATE_POOLS (1)
 #define LWIP_PBUF_FROM_CUSTOM_POOLS (0)
 #define MEMP_USE_CUSTOM_POOLS (0)
-#define PBUF_POOL_SIZE (6)
+#define PBUF_POOL_SIZE (5)
```

Comment:
This seems pretty small to me, but maybe it's ok....  Need to support at least incoming message, outgoing message, retransmit buffer, standalone ack.  That's for a single message exchange.  And assuming that nothing else uses pbufs.

I suspect @tcarmelveilleux has thoughts on this.